src/components-examples/cdk-experimental/menu/cdk-menu-context/cdk-menu-context-example.css

```diff
@@ -0,0 +1,33 @@
+.cdk-menu {
```

Comment:
For the examples, we try to prefix all of the styles with `.example-` so it's clear where they're coming from

Reply:
Those are the classes that the cdk directives add.

Reply:
Yeah, we can either add additional class in the example, or put all of these as child selectors, e.g.
```css
.example-menu { ... }
.example-menuitem { ...}

/** OR **/
.example-menu .cdk-menu {
  ...
}

.example-menu .cdk-menuitem {
  ...
}

```

Reply:
Ah, ok. I see what you were saying now.

Reply:
@jelbourn Done